link/tracepoint_test.go

```diff
@@ -48,6 +48,19 @@ func TestTracepoint(t *testing.T) {
 	}
 }
 
+func TestNonexistantTracepoint(t *testing.T) {
```

Comment:
Could this be `TestTracepointMissing` instead? I used `-run="Test(Kprobe|Kretprobe|Trace)"` frequently for this package.